src/Sarif.Viewer.VisualStudio.ResultSources.Domain.Core/Services/IResultSourceService.cs

```diff
@@ -20,6 +20,12 @@ public interface IResultSourceService
         /// </summary>
         event EventHandler<ResultsUpdatedEventArgs> ResultsUpdated;
 
+        /// <summary>
+        /// Gets a value indicating whether this service is active in the project.
+        /// </summary>
+        /// <returns>True if the service is active; otherwise, false.</returns>
+        Task<bool> IsActiveAsync();
```

Comment:
>Task<bool> IsActiveAsync(); [](http://example.com/codeflow?start=8&length=27)

I see this is a added method in the interface, is it used by external, is it a breaking change

Reply:
I don't think we need to worry, since the feature hasn't been completely checked in.